docker-compose.override.yml

```diff
@@ -20,8 +20,15 @@ services:
     depends_on:
       - watch-gulp
   ui:
+    build:
+      context: .
+      target: gulp
+    command: npx gulp server
+    ports:
+      - 8889:80
```

Comment:
`8889:80` is already inherited from `docker-compose.yml`?

Reply:
(is it?)

Reply:
Have to add in `3000`, so we have to redeclare it.